dm/master/server.go

```diff
@@ -289,7 +289,7 @@ func (s *Server) RegisterWorker(ctx context.Context, req *pb.RegisterWorkerReque
 		}
 	}
 	fmt.Println("=======Register a worker")
-	s.coordinator.AddWorker(req.Name, req.Address)
+	s.coordinator.AddWorker(req.Name, req.Address, nil)
```

Comment:
Why do we pass a `nil` client here? or when do we need a non-`nil` client? (just for testing?)

Reply:
Yes. Only for testing.